feat: FIT-585: Remove schema from members column

mcanu opened the pull request:
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

mcanu committed:
Remove schema from members columns if flag activated

yyassi-heartex commented on label_studio/data_manager/functions.py:
```diff
@@ -69,6 +70,10 @@ def get_all_columns(project, *_):
         i += 1
 
+    remove_members_schema = flag_set(
+        'fflag_all_feat_utc_204_users_performance_improvements_in_dm_for_large_orgs', user='auto'
```
I think it should be `fflag_feat_fit_449_datamanager_filter_members_short` because the FE is checking for that FF, not `fflag_all_feat_utc_204_users_performance_improvements_in_dm_for_large_orgs`
```suggestion
        'fflag_feat_fit_449_datamanager_filter_members_short', user='auto'
```

yyassi-heartex replied:
though technically both those FFs should have the same state since they work closely together

mcanu committed:
Update label_studio/data_manager/functions.py

Co-authored-by: Yousif Yassi <104568407+yyassi-heartex@users.noreply.github.com>

mcanu commented:
/fmt
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17438626217)

mcanu committed:
Apply pre-commit linters

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17438626217

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17441643492)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17441643492

mcanu committed:
Merge branch 'develop' into 'fb-fit-585'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17441643492

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17467558225)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17467558225

mcanu committed:
Merge branch 'develop' into 'fb-fit-585'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17467558225

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17475912848)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17475912848

mcanu committed:
Merge branch 'develop' into 'fb-fit-585'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17475912848

mcanu commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17476211499)
> Successfully merged: Already up to date.
